src/test/java/uk/gov/hmcts/reform/civil/service/StateGeneratorServiceTest.java

```diff
@@ -66,53 +67,16 @@ public void shouldReturn_Awaiting_Directions_Order_Docs_WhenMakeOrderSelectedAnd
         CaseData caseData = CaseData.builder()
             .ccdState(APPLICATION_SUBMITTED_AWAITING_JUDICIAL_DECISION)
             .judicialDecision(new GAJudicialDecision(MAKE_AN_ORDER))
-            .judicialDecisionMakeOrder(GAJudicialMakeAnOrder.builder().directionsText("test").build())
+            .judicialDecisionMakeOrder(GAJudicialMakeAnOrder.builder()
+                                           .directionsText("test")
+                                           .makeAnOrder(GIVE_DIRECTIONS_WITHOUT_HEARING).build())
             .build();
 
         CaseState caseState = stateGeneratorService.getCaseStateForEndJudgeBusinessProcess(caseData);
 
         assertThat(caseState).isEqualTo(AWAITING_DIRECTIONS_ORDER_DOCS);
     }
 
-    @Test
```

Comment:
May I know the reason for removing these tests please?

Reply:
Explain that to Karthick offline